packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -125,12 +125,18 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
   const hangUpButtonProps = usePropsFor(EndCallButton);
 
   const participantsButtonStyles = useMemo(
-    () => mergeButtonBaseStyles(props.increaseFlyoutItemSize ? participantButtonWithIncreasedTouchTargets : {}),
+    () =>
+      props.increaseFlyoutItemSize
+        ? { ...participantButtonWithIncreasedTouchTargets, ...controlButtonBaseStyle }
```

Comment:
this won't do a deep merge, so if `participantButtonWithIncreasedTouchTargets` has 
```
{
  propertyY: {
    propertyA: 1
}
```
and `controlButtonBaseStyle` has:
```
{
  propertyY: {
    propertyB: 2
}
```
doing `{ ...participantButtonWithIncreasedTouchTargets, ...controlButtonBaseStyle }` would result in:
```
{
  propertyY: {
    propertyB: 2
}
```
instead of:
```
{
  propertyY: {
    propertyA: 1,
    propertyB: 2
}
```

Reply:
Fluent exposes a `merge` function, perhaps that would work instead?

Reply:
In this case, we define both `participantButtonWithIncreasedTouchTargets` and `controlButtonBaseStyle`. So if `merge` doesn't work, we could just define two different styles for the two cases (instead of trying to merge).

Actually, what we want is probably [`concatStyleSets`](https://docs.microsoft.com/en-us/javascript/api/merge-styles?view=office-ui-fabric-react-latest#@fluentui-merge-styles-concatstylesets-function(2))

Reply:
Key is to avoid registering CSS classes here.